virt/virt-2-6-release-notes.adoc

```diff
@@ -75,108 +65,95 @@ The SVVP Certification applies to:
 * You can now control which network the Containerized Data Importer (CDI) uses when importing virtual machine disk images by adding the following annotation to the `metadata.annotations` field of a `DataVolume` object:
 `k8s.v1.cni.cncf.io/networks: <network_name>`:: If you use this annotation, the transfer pod will use both the default network from the cluster and the secondary Multus network.
 
+
 [id="virt-2-6-web-new"]
 === Web console
 
-//CNV-9312: The VM wizard has been completely redesigned, simplifying VM creation
 * The virtual machine wizard has been redesigned to simplify the process of xref:../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[creating virtual machines] by using preconfigured xref:../virt/vm_templates/virt-creating-vm-template.adoc#virt-creating-vm-template[virtual machine templates].
 
 * When you xref:../virt/vm_templates/virt-creating-vm-template.adoc#virt-creating-template-wizard-web_virt-creating-vm-template[create a virtual machine template with the interactive wizard], selecting the *Operating System* will automatically select the default *Flavor* and *Workload Type* for that operating system. Virtual machines created from a virtual machine template now have these details automatically selected.
 
+
 [id="virt-2-6-changes"]
 == Notable technical changes
 
-//CNV-9339: When deploying from the GUI, the CR for the HCO is now created during subscription
 * The procedure for xref:../virt/install/installing-virt-web.adoc#installing-virt-web[installing {VirtProductName} in the web console] has been streamlined. You can now create the `HyperConverged` custom resource immediately after installing the {VirtProductName} Operator by clicking *Create HyperConverged*.
 
-//CNV-10153: BareMetalPlatform removed from HCO CR
 * Previously, there was a `spec.BareMetalPlatform` field in the `HyperConverged` object. This field has been removed.
 
-* The Containerized Data Importer (CDI) configuration parameters have moved from the `CDIConfig` object to the `CDI` object. All changes to the CDI configuration must now be made in the `spec.config` field of the `CDI` object.
+* The Containerized Data Importer configuration parameters have moved from the `CDIConfig` object to the `CDI` object. All changes to the CDI configuration must now be made in the `spec.config` field of the `CDI` object.
 
 * Istio sidecar injection to transfer pods is now disabled by default. You can override the default option by using the following annotation in the `metadata.annotations` field of a `DataVolume` object:
 `sidecar.istio.io/inject: “true”`::  Set this option to enable sidecar injection to the transfer pods.
 
 [id="virt-2-6-known-issues"]
 == Known issues
 
-//New known issues for 2.6 (WIP)
-
 * Some CDI operations are currently not xref:../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[preallocated] when requested. These include:
```

Comment:
We have a bit of inconsistency in the known issues and storage section of the declaration of 'Containerized Data Importer', 'Containerized Data Importer (CDI)', and here just plain 'CDI'.
Personally, I believe since the user almost exclusively interacts with the 'CDI' object, then after first usage (in Storage section) we could just go with CDI for all usage, however I see the value in 'Containerized Data Importer (CDI)' for each independent release note (and then CDI for remaining part of that RN) because these are references and not engaged with linearly.

Reply:
I'm going with the latter, because you're right, I doubt most people are going to be reading the RNs top to bottom